x-pack/plugins/alerting/server/authorization/alerting_authorization.ts

```diff
@@ -215,40 +197,14 @@ export class AlertingAuthorization {
          * as Privileged.
          * This check will ensure we don't accidentally let these through
          */
-        throw Boom.forbidden(
-          getUnauthorizedMessage(ruleTypeId, ScopeType.Consumer, consumer, operation, entity)
-        );
+        throw Boom.forbidden(getUnauthorizedMessage(ruleTypeId, consumer, operation, entity));
```

Comment:
By looking at the tests, if you feel the error messages are now misleading, you could change the `getUnauthorizedMessage` calls to something like this (use legacyConsumer):

```suggestion
        throw Boom.forbidden(getUnauthorizedMessage(ruleTypeId, legacyConsumer, operation, entity));
```